src/pages/Admin/Venue/TemplateCard/TemplateCard.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import classNames from "classnames";
+
+import { HIDDEN_BURN_VENUE_TEMPLATES, Template } from "settings";
+
+import "./TemplateCard.scss";
+
+export interface TemplateCardProps {
+  selected: boolean;
+  template: Template;
+  onClick: () => void;
+  thumbnail?: string;
+}
+
+export const TemplateCard: React.FC<TemplateCardProps> = ({
+  template: { template, name, description },
+  onClick,
+  selected,
+  thumbnail,
+}) => {
+  const containerClass = classNames({
+    TemplateCard: true,
```

Comment:
Helps to always have one general class at the root level named per B of BEM.

[OOS] There might be a pattern to maybe reuse React components names in a more robust manner. e.g.
```ts
const containerClass = classNames({
  [TemplateCard.name]:TemplateCard.name,
  // ....
});
```
This name will always follow whichever name the component is refactored to.